demo-notebooks/guided-demos/0_basic_ray.ipynb

```diff
@@ -61,7 +61,8 @@
     "# Create and configure our cluster object\n",
     "# The SDK will try to find the name of your default local queue based on the annotation \"kueue.x-k8s.io/default-queue\": \"true\" unless you specify the local queue manually below\n",
     "cluster = Cluster(ClusterConfiguration(\n",
-    "    name='raytest',\n",
+    "    name='raytest', \n",
+    "    head_gpus=1, # If you are using GPU, this needs to be specified \n",
```

Comment:
Perhaps wording like this would work a bit better
"For GPU enabled workloads set the `head_gpus` and `num_gpus`"